tests/tests_unit/rules/test_importers/test_inference_importer.py

```diff
@@ -46,6 +48,28 @@ def test_rdf_inference():
     assert len(InformationAnalysis(rules).multi_value_properties) == 4
 
 
+def test_rdf_inference_with_removal_of_unknown_type():
+    EX = Namespace("http://example.org/")
+    SUBSTATION = Namespace("http://example.org/substation/")
+    TERMINAL = Namespace("http://example.org/terminal/")
+    store = NeatGraphStore.from_oxi_store()
+
```

Comment:
you can see that here one of substations points to non-existing terminal, but resulting data model does not have multi-value type since Uknown type is dropped